hack/jenkins/e2e.sh

```diff
@@ -107,6 +107,7 @@ GCE_PARALLEL_SKIP_TESTS=(
     "Nodes\sNetwork"
     "Nodes\sResize"
     "MaxPods"
+    "DaemonRestart"
```

Comment:
I think this is actually a no-op, since the regex on line 92 will match as well.

it'd probably be a good idea to explicitly focus these tests in the appropriate project so they actually get run somewhere.